### PR TITLE
fix: entry point page free kolour

### PR DESCRIPTION
**Pull request by sk-tenba**

### CODE REVIEW

**Comment by sk-tenba** [comment]
figure1

**Comment by sk-tenba** [comment]
figure3

File: containers/PageKolours/containers/Menu$TopNavigation/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+type Props = {
+  className?: string;
+  style?: React.CSSProperties;
+};
+
+export default function Menu$TopNavigation({ className, style }: Props) {
+  return (
+    <div className={cx(styles.container, className)} style={style}>
+      <div className={styles.topNavContainer}>
+        <div className={styles.currentNav}>The Origin of Kolours</div>
```
Comment (sk-kitsune) [inline containers/PageKolours/containers/Menu$TopNavigation/index.tsx:17]: My suggestion is to define `Menu$TopNavigation` at `teiki-components` as:

```
type Item = { label: string; key: string; href: string; }
function Menu$TopNavigation(props: { items: Item[], activeKey: string });
```

Basically, I am trying to copy from https://ant.design/components/menu

File: containers/PageMintByColorPicker/index.module.scss [hunk 2]
```diff
@@ -1,5 +1,12 @@
 .container {
   display: block;
+  display: flex;
```
Comment (sk-kitsune) [inline containers/PageMintByColorPicker/index.module.scss:3]:
```
display: block;
display: flex;
```


?????

File: containers/PageKolours/containers/Menu$TopNavigation/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.link {
+  color: colors.$black80;
+  font-weight: 600;
+  font-size: 16px;
+}
+
+.link:hover {
+  color: colors.$black80;
```
Comment (sk-kitsune) [inline containers/PageKolours/containers/Menu$TopNavigation/index.module.scss:32]: hover and nothing changes?

File: containers/PageMintByColorPicker/index.module.scss [hunk 4]
```diff
@@ -1,5 +1,12 @@
...
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.main {
+  flex: 1 1 0;
```
Comment (sk-kitsune) [inline containers/PageMintByColorPicker/index.module.scss:9]: This part looks like too much structural changes just to make the footer panel sticks to the bottom.

I suggest a less intrusive way:

```
.main {
  min-height: calc(100vh - 200px);
}
```

**Review by sk-kitsune — CHANGES_REQUESTED** [review]
⛔ **Changes requested**: there are blocking issues.

**Comment by sk-tenba** [comment]
[Screencast from 31-03-2023 10:54:05.webm](https://user-images.githubusercontent.com/110231356/229018763-636b9357-3dab-4838-bc97-08b83866f18b.webm)

**Comment by sk-tenba** [comment]
[Screencast from 31-03-2023 10:54:25.webm](https://user-images.githubusercontent.com/110231356/229018795-81cbe3bb-611b-4da1-bdec-39a2732318d9.webm)

**Review by sk-kitsune — APPROVED** [review]
✅ **Approved**: no blocking issues

**Pull request merged**